talk-proposals/CogX-TuringWay-MS.md

```diff
@@ -0,0 +1,102 @@
+# CogX - Talk Proposal
+
+- **Abstract by:** Malvika Sharan
+- **Conference** https://cogx.co/
+- **Date:** 8th - 10th June 2020
+
+## Talk Title
+
+Building a culture of collaboration in The Turing Way
+
+## Talk Description
+
+_Please give a brief description of your talk (max 500 words)._
+_The evaluation criteria of proposals will be:_
+
+- _Expected general interest of the talk_
+- _Academic standard and global impact_
+- _Breadth of domain application_
+
+Online behaviour over the last months during the COVID-19 pandemic has shifted significantly. 
+With buzzwords like “flatten the curve” and trending hashtags on Twitter like “#StayHomeSaveLives”, we have become more inclined to stay informed about what exactly is going on in our communities. 
+Digital technologies and data-driven approaches have played vital roles during this situation in creating opportunities for us to take meaningful actions as leaders, researchers, innvovators and community members.
+Already 100s of online projects have been launched to promote the development of technical solutions that can take on medical and social challenges related to coronavirus pandemic. 
+There has never been a better time for online platforms and communities to thrive and test innovative ideas for a common social cause.
+This is particularly empowering for individuals who want to lead and involve others in data-driven projects that identify interdisciplinary applications of artificial intelligence (AI) and machine learning (ML).
+However, such efforts require understanding of project design, data management, software development, and continuous integration techniques: skills that are not widely taught to academic leaders and data scientists.
+
+In this talk, I will focus on the importance of documenting and sharing knowledge around best practices in data science that can be adopted by various stakeholder in the current crisis and in the future.
+Such resources, when collaboratively developed, can have a significant impact on improving access to information and the decision-making process by our peers and future generations.
```

Comment:
Add a sentence here that says something like "projects and initiatives must be designed for inclusion from the start"